### ISSUE TITLE
Bug Report (?): Commons does not show Images when Web-Access is disabled

### ISSUE THREAD

**Issue by Ope30** [gnosygnu/xowa#176]
Hola.

Is it normal that Commons doesn't show Images when no connection to the Internet is avaible? Not talking about images you automatically download, talking about the ones in the Wiki itself, that are there even without internet. Have a look: https://image.prntscr.com/image/MKqeQz4wTES0fjT83PxFmA.jpeg
As you can see, Commons doesn't show the Image. But when I enable Web-Access it shows it.

**Comment by Ope30** [comment]
I mean Commons actually shows images, but as you know, I always have to reload page for Summary to show. https://image.prntscr.com/image/VqCGvaobQNeWc_RKzuJg0A.jpeg
After you reload page: https://image.prntscr.com/image/TKrATVjBR4_AEqYl_RJqZA.jpeg

**Comment by gnosygnu** [comment]
> Is it normal that Commons doesn't show Images when no connection to the Internet is avaible?

Yeah, all Commons images need to be downloaded. It would take too much space to bundle the images with the XOWA Commons databases

> I mean Commons actually shows images, but as you know, I always have to reload page for Summary to show. https://image.prntscr.com/image/VqCGvaobQNeWc_RKzuJg0A.jpeg
> After you reload page: https://image.prntscr.com/image/TKrATVjBR4_AEqYl_RJqZA.jpeg

That's odd. I tried now with commons.wikimedia.org/wiki/File:Opera_Mundi_2.png . 
* It loaded the image fine the first time (Summary was there)
* Reloading the page showed both image and Summary (Image didn't go away)

I think this is still an issue with your commons download. I've been working on the Spanish wikis, but I'll try to upload 2017-08 Commons next week

**Comment by Ope30** [comment]
Hey.

Are you okay? Just asking, because you were not active for a while now. :O

**Comment by gnosygnu** [comment]
Yeah, I've been a bit lazy on my side.

I uploaded a minor release tonight, and will be uploading 2017-09 English Wikipedia later this week.

As to your request for a refresh of Commons, I guess it'll be 2017-09 now. I'll upload it after the 2017-09 enwiki.

Thanks.

**Comment by Ope30** [comment]
> Yeah, I've been a bit lazy on my side.

No problem. Glad you're back!
